webdriverio/appium-app/examples/appium-flutter-integration/test/specs/flutter.spec.ts

```diff
@@ -0,0 +1,29 @@
+// @ts-ignore
+import { browser, expect } from '@wdio/globals';
+import "jest"
+
+// @ts-ignore
+import FlutterIntergrationDriverService from 'wdio-flutter-by-service';
```

Comment:
Is this import needed even if it is not used in the code?

Reply:
only for my local compiler by default flutterByValueKey$ is not recognized,  it can be deleted